frontend/src/components/Toolbar.tsx

```diff
@@ -155,7 +156,8 @@ class Toolbar extends React.Component<ToolbarProps> {
               <div>{/* Extra level needed by tooltip when child is disabled */}
                 <BusyButton id={b.id} color='secondary' onClick={b.action} disabled={b.disabled}
                   title={b.title} icon={b.icon} busy={b.busy || false}
-                  outlined={b.outlined || false} />
+                  outlined={b.outlined && !b.primary || false}
```

Comment:
Can we add parentheses here? Off the top of my head, I don't know what this evaluates to. :)

Reply:
Good call